saleor/checkout/complete_checkout.py

```diff
@@ -598,6 +607,16 @@ def complete_checkout(
     for thread race.
     :raises ValidationError
     """
+
+    # force recalculation of taxes
+    fetch_checkout_prices_if_expired(
```

Comment:
The name of the function does not say whether it actually forces a recalculation. This comment is confusing.